project/Dependencies.scala

```diff
@@ -2,28 +2,25 @@ import sbt._
 
 object Dependencies {
   object V {
-    val log4jApi        = "2.6.2"
-    val log4jCore       = "2.6.2"
-    val netty           = "4.1.15.Final"
-    val guava           = "19.0"
-    val jacksonDatabind = "2.6.7"
-    val commonsPool     = "2.4.2"
-    // (test only)
-    val junitInterface  = "0.8"
-    val nettyTcnative   = "2.0.6.Final"
+    val log4j           = "2.13.3"
+    val netty           = "4.1.51.Final"
+    val guava           = "24.1.1-jre"
```

Comment:
I _think_ it makes sense to go as far as `28.1-jre` as its the version we use in enrich.